artifactory/cli.go

```diff
@@ -1076,6 +1090,31 @@ func getReplicationDeleteFlags() []cli.Flag {
 	return getRepoDeleteFlags()
 }
 
+func getTokenCreateFlags() []cli.Flag {
+	return append(getServerWithClientCertsFlags(), []cli.Flag{
+		cli.StringFlag{
+			Name:  "groups",
+			Usage: "[Optional] A list of comma-separated groups that the token is associated with.` `",
+		},
+		cli.StringFlag{
+			Name:  "admin-privileges",
+			Usage: "[Optional] An Artifactory instance ID to be given admin privileges.` `",
+		},
+		cli.StringFlag{
+			Name:  "expiry",
+			Usage: "[Default: " + strconv.Itoa(cliutils.TokenExpiry) + "] The time in seconds for which the token will be valid.` `",
+		},
+		cli.BoolFlag{
+			Name:  "refreshable",
+			Usage: "[Default: false] Set to true if you'd like the the token to be refreshable. A refresh token will also be returned in order to be used to generate a new token once it expires.` `",
+		},
+		cli.StringFlag{
+			Name:  "audience",
+			Usage: "[Optional] A space-separate list of the other Artifactory instances or services that should accept this token.` `",
```

Comment:
I suggest changing this description to the following:
[Optional] A space-separate list of the other Artifactory instances or services that should accept this token identified by their Artifactory Service IDs, as obtained by the 'jfrog rt curl api/system/service_id' command.